locksmith/src/operations/wedlocksOperations.ts

```diff
@@ -424,6 +425,13 @@ export const notifyNewKeyToWedlocks = async (key: Key, network: number) => {
   const withLockImage = (customContent || '')?.length > 0
   const lockImage = `${config.services.locksmith}/lock/${lockAddress}/icon`
 
+  const [transactionsHash] = key?.transactionsHash ?? []
```

Comment:
This email is sent for the _latest_ transaction so let's only send the latest transaction hash... because that is the one for this receipt.

Reply:
actually, i was extracting the first transaction hash from the array `[transactionsHash] = key?.transactionsHash ?? []`,

changed to the the last transaction hash instead